src/devtools/views/Element.js

```diff
@@ -45,6 +52,13 @@ export default function ElementView({ index, style }: Props) {
   const showDollarR =
     isSelected && (type === ElementTypeClass || type === ElementTypeFunction);
 
+  const component = useRef(null);
+  useEffect(() => {
+    if (isSelected && component.current !== null) {
+      component.current.scrollIntoView();
```

Comment:
It sometimes looks nice with `{ behavior: 'smooth' }`, sometimes looks jarring 🤔